cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js

```diff
@@ -362,6 +362,28 @@ server.get('GetPaymentMethods', server.middleware.https, function (req, res, nex
     return next();
 });
 
+server.get('GetCheckoutPaymentMethods', server.middleware.https, function (req, res, next) {
```

Comment:
did we not have this already for showing the local payment methods ?

Reply:
@KadoBOT @zaiddreakh are we not doing this call twice now one for cards and one for alternative payment methods ?

Reply:
@rikterbeek Looks like we are, will refactor this for now and tomorrow will see what Bas thinks about this

Reply:
Yes, we only render the card component with this PR. This required the full /paymentMethods response on the checkout component. 
Next to that, the LPM's will be rendered the current way still. 
This means that there are 2 calls, but one will be removed after Generic Component is integrated for LPM's